### PR TITLE
chore(ckbtc): Upgrade ckBTC ledger suite

### PR DESCRIPTION
**Pull request by gregorydemay**
Proposals to upgrade the ckBTC ledger suite (index, ledger and archive canisters) to [ledger-suite-icrc-2024-11-28](https://github.com/dfinity/ic/releases/tag/ledger-suite-icrc-2024-11-28).

### CODE REVIEW

File: rs/bitcoin/ckbtc/mainnet/ckbtc_archive_upgrade_2024_12_09.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Proposal to upgrade the ckBTC archive canister
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `2190613d3b5bcd9b74c382b22d151580b8ac271a`
+
+New compressed Wasm hash: `f94cf1db965b7042197e5894fef54f5f413bb2ebc607ff0fb59c9d4dfd3babea`
+
+Upgrade args hash: `0fee102bd16b053022b69f2c65fd5e2f41d150ce9c214ac8731cfaf496ebda4e`
```
Comment (lpahlavi) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_archive_upgrade_2024_12_09.md:9]: verified!

File: rs/bitcoin/ckbtc/mainnet/ckbtc_archive_upgrade_2024_12_09.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Proposal to upgrade the ckBTC archive canister
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `2190613d3b5bcd9b74c382b22d151580b8ac271a`
+
+New compressed Wasm hash: `f94cf1db965b7042197e5894fef54f5f413bb2ebc607ff0fb59c9d4dfd3babea`
```
Comment (lpahlavi) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_archive_upgrade_2024_12_09.md:7]: verified!

File: rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_12_09.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Proposal to upgrade the ckBTC index canister
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `2190613d3b5bcd9b74c382b22d151580b8ac271a`
+
+New compressed Wasm hash: `2adc74fe5667f26ea4c4006309d99b1dfa71787aa43a5c168cb08ec725677996`
+
+Upgrade args hash: `0fee102bd16b053022b69f2c65fd5e2f41d150ce9c214ac8731cfaf496ebda4e`
```
Comment (lpahlavi) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_12_09.md:9]: verified!

File: rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_12_09.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+# Proposal to upgrade the ckBTC ledger canister
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `2190613d3b5bcd9b74c382b22d151580b8ac271a`
+
+New compressed Wasm hash: `25071c2c55ad4571293e00d8e277f442aec7aed88109743ac52df3125209ff45`
+
+Upgrade args hash: `0fee102bd16b053022b69f2c65fd5e2f41d150ce9c214ac8731cfaf496ebda4e`
```
Comment (lpahlavi) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_12_09.md:9]: verified!

File: rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_12_09.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+# Proposal to upgrade the ckBTC index canister
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `2190613d3b5bcd9b74c382b22d151580b8ac271a`
+
+New compressed Wasm hash: `2adc74fe5667f26ea4c4006309d99b1dfa71787aa43a5c168cb08ec725677996`
```
Comment (lpahlavi) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_12_09.md:7]: verified!

File: rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_12_09.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
+# Proposal to upgrade the ckBTC ledger canister
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `2190613d3b5bcd9b74c382b22d151580b8ac271a`
+
+New compressed Wasm hash: `25071c2c55ad4571293e00d8e277f442aec7aed88109743ac52df3125209ff45`
```
Comment (lpahlavi) [inline rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_12_09.md:7]: verified!

**Review by lpahlavi — APPROVED** [review]

File: rs/bitcoin/ckbtc/mainnet/ckbtc_archive_upgrade_2024_12_09.md (hunk 2 above)
Comment (THLO) [reply rs/bitcoin/ckbtc/mainnet/ckbtc_archive_upgrade_2024_12_09.md:7]: Verified!

File: rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_12_09.md (hunk 5 above)
Comment (THLO) [reply rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_12_09.md:7]: Verified!

File: rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_12_09.md (hunk 6 above)
Comment (THLO) [reply rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_12_09.md:7]: Verified!

File: rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_12_09.md (hunk 3 above)
Comment (THLO) [reply rs/bitcoin/ckbtc/mainnet/ckbtc_index_upgrade_2024_12_09.md:9]: Verified!

File: rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_12_09.md (hunk 4 above)
Comment (THLO) [reply rs/bitcoin/ckbtc/mainnet/ckbtc_ledger_upgrade_2024_12_09.md:9]: Verified!

**Review by THLO — APPROVED** [review]

**Pull request merged**